graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetConfig.java

```diff
@@ -255,7 +256,11 @@ public boolean isRegularIndex() {
     @JsonIgnore
     public boolean canHaveCustomFieldMappings() {
         final String indexTemplateType = this.indexTemplateType().orElse(null);
-        if (EVENT_TEMPLATE_TYPE.equals(indexTemplateType) || "failures".equals(indexTemplateType)) {
+        if (EVENT_TEMPLATE_TYPE.equals(indexTemplateType) ||
```

Comment:
Would it make sense to move the template types to a set and test presence in a set?